readme.md

```diff
@@ -12,7 +12,7 @@ The Official AMP Plugin, supported by the AMP team. Formerly Accelerated Mobile
 **License:** [GPLv2 or later](http://www.gnu.org/licenses/gpl-2.0.html)  
 **Requires PHP:** 5.6  
 
-[![Build Status](https://travis-ci.org/ampproject/amp-wp.svg?branch=develop)](https://travis-ci.org/ampproject/amp-wp) [![Coverage Status](https://img.shields.io/codecov/c/github/ampproject/amp-wp/develop.svg)](https://codecov.io/gh/ampproject/amp-wp) [![Built with Grunt](https://gruntjs.com/cdn/builtwith.svg)](http://gruntjs.com) 
+[![Build Status](https://github.com/ampproject/amp-wp/workflows/Build,%20test%20&%20measure/badge.svg)](https://github.com/ampproject/amp-wp/actions?query=branch%3Adevelop+workflow%3A%22Build%2C+test+%26+measure%22) [![Coverage Status](https://img.shields.io/codecov/c/github/ampproject/amp-wp/develop.svg)](https://codecov.io/gh/ampproject/amp-wp) [![Built with Grunt](https://gruntjs.com/cdn/builtwith.svg)](http://gruntjs.com)
```

Comment:
At the moment the badge label is "Build, test & measure". Customizing the label is not possible but it has been reported on GitHub's community forums as a [requested feature](https://github.community/t/let-us-configure-the-badge-labels/138418).

![image](https://user-images.githubusercontent.com/16200219/98453973-008c5400-2157-11eb-8bf3-aab9a7bf4e8d.png)

Reply:
Also, the status is for the default branch (`develop`).